Integrations/integration-Threat_Grid.yml

```diff
@@ -0,0 +1,922 @@
C5EkjRK9ypS/ZMgHmUU0k45so76x8AL8Jm4JEmd48VfkqTuWAC8moyvJ7iJcUiStPPsDrfd7lCn+hsBBxiFpB1id7hRSMDxVVQ/I7iTcUjavvO1TcvmwALgJFL2OWA/45B2yTxIry4mJr8I7G8ckiR1hwVxkiR1w53zqU1nQfg2nCRJGpa/D+ozgb2MQpLG3kHFmslvRsTbIS00DkkamdsWU5v+l5SeiSs7SLPp6KKInwEPMwpJkrrBB+KSJLVbH3gt8DW7wkmSNDvsDrddHpAifRSYbxSSdpjd4YbthCovfwrc0Sgk7dj52u5wsygHXhLwbeBg45DmQuyRIn0YeA+wm3lIktRuFsRJktReE8VU8Q0iPc9rviRJGqL7J+qPMVOYL0kaX0uBDwfpbXaFk6SROrCYmvwGKV4B9IxDmlsBj8n75U+AO5uGJEnt5cNxSZLa6f5Fnv8PcHujkCRp9tgd7nrdK6X4KBbDSdpZdocbltXFoP5uJJcNk7Sz52u7w82S+5VV9mPgSKOQhifBCuBM4ERcnliSpFayIE6SpHbpAa9LKT4F7GkckiRpiI4mZZ/EZVIladw9uU71j4HVRiFJI5MBL08RZ0DsYRzSSOTAS4s1az/KTOdcSZLUIoURSJLUGvsX/SUfAu5oFJIkzT67w23TYQQfw2I4SbvC7nBzbTHw9kjxSKOQtGvna7vD7aK9iol174e4l1FI4yA9uBjE4cCDgbPMQ5KkdrBDnCRJ7XCfoqh/gsVwkiRp+CbqPP8CsLtRSNLYulmxsPhRkCyGk6TRunWvzn5sMZw0bmJlnbLvAA83C0mS2sGCOEmSmu/4FHwK2MsoJEmaG3aHu067FXl+BrCfUUjatVmN3eHm0H1qsm8CBxuFpF0/X9sdbhccG5F9M2DCKKQxFCxO8EHg7UDfQCRJajYL4iRJaq4+8K6IeDvhMuiSJGnoekW//BRwM6OQpLH1z6nm08BuRiFJI5MDb07BacAC45DGW0ocX0z1vgDsYRqSJDWXD88lSWqmvYqNxUdJcTejkCRpCPo2Yr1Cr56mzgNIrye4i4lI2mV2h5sL84D/ItJxRiFp9s7XdofbCYuLqbUfgPQAo5CaJI7KB/N+QKrvB0yax5DTf6gZSJJ2nR3iJElqnlVFnX2HhMVwkiQNwSBcKeVaPCoFTzcGSRpLexVLqi8kOM4oJGmk9isW9r5mMZzUTCk4qKqz7wB3Ng1JkprHgjhJkprlXlWd/QBYZRSSJGlEbp6CU4xB0qywO9xsu1leZz+CdFejkDS752u7w+3o+biX8V0IL3RSs+0FfAl4pFFIktQsFsRJktQcTyZlnwF2MwpJkobD7nB/Y/c64xOQFhqFJI2do2uybydYYRSSNFL3qcm+HaTlRiG1wrxEvA94gVFIktQchRFIktQIL4oUrzQGSZKGbP0KMwC40bnkeUWWpbdGcKCBSJoVdoebTQ8meD8wzygkzf752u5wO+CEFLwJn79JbZMgvRq4CfBUoDQSSZLGmx3iJEka+4k2/w+wGE6SJI3aoyN4hDFI0tg5NhEfwmI4SRq1FwTpbYTFcFJbJXhSMbHsdGC+aUiSNN4siJMkaXzl+WmTp0B6vlFIkjR8g/NXGcKV9k8pO9kYJM0au8PNlmelYI3FF5Lm7nxtd7jt9IKt3aMktV56QBHTpwMLzEKSpPFlQZwkSeNpfjE1+bEUPNEoJEnSiGVFUb8fYg+jkKSxkYDXBOmNW/9ZkjS68/EbLYaTOvfNv08xdflngSWGIUnSePLNQUmSxs/igunTgXsYhSRJIzRxthkAvSieGnAXk5A0a+wOt6vyfM3k20g8ySgkze352u5w1yPLT5t8O/hCq9RNcVSRp6+WxL2BP5mHJEljNlg3AkmSxsqiYmHxGSyGkyRppAZRGsKMG0biFcYgSWMj601NnpqwGE6SRizvTUy+29UdpI4LblPU2TeAZYYhSdJ4sUOcJEnjY1FRzf8ccGejkCRpxDb0Ox9BsbgkkZ0Uid09ICTNGrvD7Yosn5h8V8BjjELS3J+v7Q63Db1i4+SHI/EPRiEJWF2U+VdL4mjgIuOQJGk82CFOkqTxsKCo5n8Ki+EkSdL4uFckHmYMkjQWUr5i3ZsTHGcUkjRSea+/5D1YDCfp6g4psnQmsK9RSJI0HiyIkyRp9OYXU5v+G7i7UUiSNHqDy5wqA/MiZW8xBkmzyu5wOyvlp617S4p4ilFIGs752u5w13k+npp8Z8DDjULStVhV5IPPgV3WJUkaBy6ZKknSaPWLNP1RgnsYhSRJY2KJU+WC6acBN/FgkDSrPnq2GVyXS+dDVtMvSupUkCIIaiAB8RpIFsNJ0jCuU5fOZ/6CAVWVqFOQBURkkALgddipU9K2JG5ZMP25Eu4JbDAQSZJGx9feJUkanV6xsfdRgvsbhSRJGiO7Ay8yBkkaC6+C9HxjkKRxOB/zXGOQtB3uUGTpk8B8o5AkaXR87V2SpNHIev0l7wni741CkqQxE1WnN79I9QuBvTwQJM2qPRaYwY77V+CFxiBpqFwu9dq8xPOxpB109yJNf7SEY4CBcUiSNHwWxEmSNFtWX8evbzyXeXWi7lXUdUZGEKl+TUR6uKFJkjReBnXe9QgOgHimR4IkjdxxkF5mDJI0cs8mxSuMQdIOC+7f6y153wAeCVQGIknScFkQJ0nS8D2DSM8zBkmSxlDa0unNz8lfDMk2TpJml93hdtQ9gzgFSEYhaajsDndNDwvS641B0s6KxMOKjWt/l7LsWXVdk+WJQZVDymH9CgOSJGkOZUYgSdJQPSCINxiDJEkaQzdMpMcagySN1M2I+iNAzygkaaRum4JTsThZ0q5K6ZmAndglSRoyO8RJkjQ8t0tUH5p5/UuSJI2l6fmd3fSiP3guMN+DQNKssjvcjti/TtlnCXYzCklDZ3e4qzow4NOQFhqFpNkQEW8A1gP/bRqSJA2HHeIkSRqOg6L2RpokSeNssKXT74ztBul4jwJJG
```

Comment:
same - we don't understand it enough now... we should either consult customers or let Lior define it